packages/controllers/src/snaps/json-schemas/SnapManifest.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * MUST be a valid SemVer version string and equal to the corresponding `package.json` field.
+ *
+ */
+type StringGyS0TG81 = string;
+/**
+ *
+ * MUST be a non-empty string less than or equal to 280 characters. A short description of the Snap.
+ *
+ */
+type StringWm7QQAOA = string;
+/**
+ *
+ * MUST be a string less than or equal to 214 characters. The Snap author's proposed name for the Snap. The Snap host application may display this name unmodified in its user interface. The proposed name SHOULD be human-readable.
+ *
+ */
+type StringDQmAWJyy = string;
+type StringG66CDL2Y = "git";
+type StringDj4X5WuP = string;
+/**
+ *
+ * MAY be omitted. If present, MUST be equal to the corresponding package.json field.
+ *
+ */
+type ObjectOfStringDj4X5WuPStringG66CDL2Y3ZIYi6Gi = {
```

Comment:
adding `title` to the schema will make these names better if we care about them. But we aren't exporting them anymore so it's probably fine to just leave them with their generated names.

Reply:
I added `title` for top-level properties but didn't bother with deeper ones. It's a little tedious!